package.json

```diff
@@ -44,11 +44,13 @@
   },
   "homepage": "https://github.com/mixpanel/panel",
   "dependencies": {
+    "core-js": "3.24.1",
```

Comment:
This and `regenerator-runtime` are only needed for running test environment, so move them to `devDependencies`